src/definitions.jl

```diff
@@ -419,8 +419,34 @@ Base.copy(x::Frequencies) = x
     fftfreq(n, fs=1)
 Return the discrete Fourier transform (DFT) sample frequencies for a DFT of length `n`. The returned
 `Frequencies` object is an `AbstractVector` containing the frequency
-bin centers at every sample point. `fs` is the sample rate of the
-input signal.
+bin centers at every sample point. `fs` is the sampling rate of the
+input signal, which is the reciprocal of the sample spacing.
+
+Given a window of length `n` and a sampling rate `fs`, the frequencies returned are 
+
+```julia
+[0, 1, ..., n/2 - 1, -n/2, ..., -1]  * fs/n   # if n is even
+[0, 1, ..., (n-1)/2, -(n-1)/2, ..., -1]  * fs/n  # if n is odd
```

Comment:
If this is Julia code, you need to make it valid Julia code: `[0:n÷2-1; -n÷2:-1] * fs/n` etcetera.  If you want to use math notation, use a LaTeX block.

Reply:
I've fixed this to use valid Julia code, thanks for the suggestion